modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -6,7 +6,8 @@
 [id='preparing-the-bare-metal-node_{context}']
 = Preparing the bare metal node
 
-Expanding the cluster requires a DHCP server. Each node must have a DHCP reservation.
+To expand your cluster, you must provide the node with the relevant IP address. This can be done with a static configuration, or with a DHCP (Dynamic host Configuration protocol) server. When expanding the cluster using a DHCP server, each node must have a DHCP reservation.
```

Comment:
```suggestion
To expand your cluster, you must provide the node with the relevant IP address. This can be done with a static configuration or with a DHCP (Dynamic Host Configuration protocol) server. When expanding the cluster using a DHCP server, each node must have a DHCP reservation.
```

Reply:
@MirzWeiss @kalexand-rh Protocol should also be capitalized. See https://www.rfc-editor.org/rfc/rfc1531